men_team_1/db/migrate/20181109011841_add_references_to_pairing.rb

```diff
@@ -0,0 +1,8 @@
+class AddReferencesToPairing < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :pairings, :mentor, index: true
+    add_foreign_key :pairings, :users, column: :mentor_id
+    add_reference :pairings, :mentee, index: true
+    add_foreign_key :pairings, :users, column: :mentee_id
+  end
+end
```

Comment:
This conflicts with the first migration. You've already created references with `foreign_key: true` so this migration is redundant in its entirety. Delete this file.